### ISSUE TITLE
Directions link in Facility Locator should not open in a new tab

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#11814]
## Issue Description
Currently in the Facility Locator the "Get Directions on Google Maps" links all open in a new tab.  This can be particularly frustrating if unanticipated by assistive tech users.

In order to avoid frustration and confusion, as well as to match the standard pattern of other Directions links throughout VA.gov, it is recommended to change the behavior of the Facility Locator Directions links from opening a new tab for Google maps to opening Google maps within the same tab.

Also, after opening the link to Google Maps, the back button should return users to the same search results state.

### Screenshot
![image](https://user-images.githubusercontent.com/106678594/223141656-38382d91-c9fa-41c6-a576-adca0c4c3449.png)

## Acceptance Criteria
- [ ] When a Veteran clicks on the Directions link from a search result, Google Maps opens in the same tab. (link has a target BLANK that can be removed)
- [ ] When a Veteran clicks on the back button from Google maps, users should be returned to the same search results state (URL is updated when the search is made)
- [ ] I do not expect that browser specific behavior is explored or implemented

**Comment by maxx1128** [comment]
I'd estimate this as a **1,** since changing this behavior is a simple HTML change of removing `target="_blank"` from the link.
